qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -202,18 +205,20 @@ def _to_program(self, response: Dict) -> RuntimeProgram:
     def run(
             self,
             program_id: str,
-            options: Dict,
+            backend_name: str,
```

Comment:
While `backend_name` is the only option right now, we specifically made the parameter `options` (a dictionary) for future expansion :)